handler.go

```diff
@@ -32,7 +31,22 @@ var (
 //		assuming that there's no package with the given name.
 //	GET /<name>/<subpkg>
 //		Package page for the given subpackage.
-func CreateHandler(config *Config) http.Handler {
+func CreateHandler(config *Config, templates *template.Template) (http.Handler, error) {
+	indexTemplate := templates.Lookup("index.html")
+	if indexTemplate == nil {
+		return nil, errors.New("template index.html is missing")
+	}
+
+	notFoundTemplate := templates.Lookup("404.html")
+	if indexTemplate == nil {
+		return nil, errors.New("template 404.html is missing")
+	}
+
+	packageTemplate := templates.Lookup("package.html")
+	if indexTemplate == nil {
+		return nil, errors.New("template package.html is missing")
+	}
```

Comment:
ℹ️ This errors should **never** happen, as the default templates will always be there.